h5c/vic/src/vic-webapp/src/app/create-vch-wizard/general/general.html

```diff
@@ -217,8 +217,7 @@
       <clr-signpost>
         <clr-signpost-content [clrPosition]="'top-left'" *clrIfOpen>
             <span>
-              Deploy the VCH with more verbose levels of logging, and optionally modify the behavior of vic-machine
-              for troubleshooting purposes.
+              Configure the VCH to send logs to a syslog endpoint that is not located in the appliance.
```

Comment:
I think the use of "appliance" here is confusing; from a VI Admin point of view, "appliance" means the OVA, not the VCH.

Reply:
@zjs @stuclem what should the wording be here?

Reply:
Agreed with @zjs. How about "Configure the VCH to send logs to a syslog endpoint that is not located in the VCH."